code/modules/admin/verbs/request_internet_sound.dm

```diff
@@ -0,0 +1,39 @@
+/mob/verb/request_internet_sound()
+	set category = "OOC"
+	set name = "Request Internet Sound"
+
+	if(GLOB.say_disabled) //This is here to try to identify lag problems
+		to_chat(usr, span_danger("Speech is currently admin-disabled."), confidential = TRUE)
+		return
+
+	if (!CONFIG_GET(flag/request_internet_sound))
+		to_chat(usr, span_danger("This server has disabled internet sound requests."), confidential = TRUE)
+		return
+
+	var/request_url = tgui_input_text(usr.client, "Please Input a URL", "Only certian sites are allowed, such as youtube, soundcloud, and bandcamp", "")
```

Comment:
```suggestion
	var/request_url = tgui_input_text(usr, "Please Input a URL", "Only certain sites are allowed, such as YouTube, SoundCloud, and Bandcamp.", "")
```

We don't need to pass in `usr.client`, just `usr` will be fine in this context. Also Grammar and Formatting nits.